api/python/provisioner/commands/confstore_export.py

```diff
@@ -54,6 +55,24 @@ class ConfStoreExport(CommandParserFillerMixin):
     input_type: Type[inputs.NoParams] = inputs.NoParams
     description = "Export pillar template data to ConfStore"
 
+    def _encrypt_value(self, key, value):
+        from cortx.utils.security import cipher
+        if not value:
+            return value
+        cypher_name = key.split('>')[0]
+        if 'bmc' in key:
```

Comment:
I suggest here to use constants for all strings like: `bmc`, `storage_enclosure`, `cluster_id`, etc.

Reply:
Will be resolved in https://jts.seagate.com/browse/EOS-19150